ui/jobs/jobs_config.js

```diff
@@ -43,7 +43,7 @@ UserConfig.registerOptions('jobs', {
       name: {
         en: 'Hide cheese icon when food > time (in seconds)',
         de: 'Verstecke das Käse Icon wenn Bufffood > Zeit (in Sekunden)',
-        fr: 'Masquer l\'icône quand Repu > durée (en secondes)',
+        fr: 'Masquer l\'icône du fromage lorsque vous mangez > durée (en secondes)',
```

Comment:
This does not match the english meaning.

Reply:
Ba en fait, il y a vraiment une icone fromage, et repu arrive quand on mange.

Reply:
Ah yes, l'option 2 est parfaite !